src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -83,7 +113,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because the New Relic NRQL violation target entity names, and NRQL query are the same respectively. Does not apply to REST source. Relevant incidents from the same [NRQL alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) will be identified. It helps relate incidents like same transaction query latency deviation, etc.
+        Correlation activated because the New Relic NRQL violation target entity names, and NRQL query are the same respectively. Does not apply to REST source. Relevant events from the same [NRQL alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) will be identified. It helps relate issues like same transaction query latency deviation, etc.
```

Comment:
we don't like to use 'etc', so we could maybe do "relate issues (for example, issues with the same transaction query latency deviation)"

Reply:
done